Tools/autotest/sim_vehicle.py

```diff
@@ -851,6 +852,11 @@ def start_mavproxy(opts, stuff):
     location = find_location_by_name(find_autotest_dir(), cmd_opts.location)
     progress("Starting up at %s (%s)" % (location, cmd_opts.location))
 
+if cmd_opts.use_dir is not None:
+    new_dir = cmd_opts.use_dir
+    os.makedirs(new_dir)
```

Comment:
Two comments from documentation:

- "Raises an error exception if the leaf directory already exists or cannot be created", so we should check for directory existence first
- "makedirs() will become confused if the path elements to create include os.pardir", pardir is .. which is very likely to be used here so we should get the absolute path first